site/styles/_shift72.scss

```diff
@@ -278,13 +282,18 @@
 
 s72-useraccount-form {
   .s72-account-password-form-group,
-  .s72-account-pin-form-group {
+  .s72-account-pin-form-group,
+  .s72-account-request-delete-form-group {
     margin-top: 30px;
 
     label {
       font-weight: $font-weight-bold;
     }
   }
+
+  .s72-account-request-delete-message {
+    padding-bottom: 0.5rem;
```

Comment:
Is `0.5em` a standard spacing amount?

Reply:
Was the amount that was used for the pin text spacing you see in the first screenshot, so I thought I should keep it consistent. Have no idea if that was just thrown in there or not.

Reply:
https://github.com/shift72/core-template/blob/develop/site/styles/_pin-codes.scss#L1